percy/snapshot.py

```diff
@@ -113,6 +116,9 @@ def percy_automate_screenshot(driver, name, options = None, **kwargs):
             options['consider_region_selenium_elements'] = options['considerRegionSeleniumElements']
             options.pop('considerRegionSeleniumElements')
 
+        if 'sync' not in options:
+            options['sync'] = False
```

Comment:
Don't set default value since in global config user can pass sync true which will not be honoured because of this